### PR TITLE
Add card popovers

### PR DESCRIPTION
**Pull request by manuthecoder**
Closes #6832

This PR adds popovers to cards!
image

### CODE REVIEW

File: app/views/stripe_cards/show.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <%= render partial: "canceled_warning", locals: { stripe_card: @card } %>
 
 <turbo-frame id="details">
-  <article class="check--form flex items-start justify-center mt3 mb3" style="gap: 16px;">
+  <article class="check--form flex <%= params[:frame] ? "flex-col" : "" %> items-start justify-center mt3 mb3" style="gap: 16px;">
```
Comment (malted) [inline app/views/stripe_cards/show.html.erb]:
```suggestion
  <article class="check--form flex <%= "flex-col" if params[:frame] %> items-start justify-center mt3 mb3 gap-4">
```

**Review by malted — DISMISSED** [review]
awesome pr!

File: app/views/stripe_cards/show.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% title @card.activated ? "Card #{@card.last_four}" : "Inactive card" %>
-<%= render "events/nav", selected: :cards %>
+<%= render "events/nav", selected: :cards if !params[:frame] %>
```
Comment (sampoder) [inline app/views/stripe_cards/show.html.erb]:
```suggestion
<%= render "events/nav", selected: :cards unless params[:frame] %>
```

File: app/views/stripe_cards/_stripe_card.html.erb [hunk 3]
```diff
@@ -33,3 +33,12 @@
   </div>
 
 </a>
+
+<% if !params[:frame] %>
```
Comment (sampoder) [inline app/views/stripe_cards/_stripe_card.html.erb]:
```suggestion
<% unless params[:frame] %>
```

File: app/views/stripe_cards/_stripe_card.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<a class="text-decoration-none" href="<%= !show_mock_data? ? (local_assigns[:href] || stripe_card_path(stripe_card)) : "javascript:;" %>">
+<a class="text-decoration-none" <% if !params[:frame] %>data-behavior="modal_trigger" data-modal="card_popover_<%= stripe_card.id %>" data-turbo-frame="_top" <% end %>href="<%= !show_mock_data? ? (local_assigns[:href] || stripe_card_path(stripe_card)) : "javascript:;" %>">
```
Comment (sampoder) [inline app/views/stripe_cards/_stripe_card.html.erb]:
```suggestion
<a class="text-decoration-none" <% unless params[:frame] %> data-behavior="modal_trigger" data-modal="card_popover_<%= stripe_card.id %>" data-turbo-frame="_top" <% end %> href="<%= !show_mock_data? ? (local_assigns[:href] || stripe_card_path(stripe_card)) : "javascript:;" %>">
```

**Review by sampoder — COMMENTED** [review]
I'd recommend using `unless` instead of `if !`

**Review by sampoder — APPROVED** [review]

**Pull request merged**